src/leap/mail/imap/mailbox.py

```diff
@@ -500,9 +500,9 @@ def fetch(self, messages_asked, uid):
         is_sequence = True if uid == 0 else False
 
         # XXX DEBUG ---  if you attempt to use the `getmail` utility under
-        # imap/tests, it will choke until we implement sequence numbers. This
-        # is an easy hack meanwhile.
-        # is_sequence = False
+        # imap/tests, or muas like mutt, it will choke until we implement
+        # sequence numbers. This is an easy hack meanwhile.
+        is_sequence = False
```

Comment:
This is ugly. Do we really need to hardcode `is_sequence` to False? In this case, can we at least remove (or comment) the previous set of `is_sequence`?

I see is a temporary fix and I will merge it like that, but I'm still not really happy to include nasty workarounds that might pile up in the future.